client/my-sites/email/email-management/home/email-plan-warnings.jsx

```diff
@@ -36,26 +30,15 @@ class EmailPlanWarnings extends React.Component {
 	renderCTAForTitanUnusedMailboxes() {
 		const { currentRoute, domain, selectedSiteSlug, translate } = this.props;
 
-		const showExternalControlPanelLink = ! isEnabled( 'titan/iframe-control-panel' );
-		const controlPanelUrl = showExternalControlPanelLink
-			? emailManagementTitanControlPanelRedirect( selectedSiteSlug, domain.name, currentRoute, {
-					context: TITAN_CONTROL_PANEL_CONTEXT_CREATE_EMAIL,
-			  } )
-			: emailManagementManageTitanAccount( selectedSiteSlug, domain.name, currentRoute, {
-					context: TITAN_CONTROL_PANEL_CONTEXT_CREATE_EMAIL,
-			  } );
+		const setupMailboxUrl = emailManagementTitanSetupMailbox(
+			selectedSiteSlug,
+			domain.name,
+			currentRoute
+		);
 
 		return (
-			<Button
-				compact
-				primary
-				href={ controlPanelUrl }
-				target={ showExternalControlPanelLink ? '_blank' : null }
-			>
-				{ translate( 'Activate Mailbox', 'Activate Mailboxes', {
-					count: getMaxTitanMailboxCount( domain ) - getConfiguredTitanMailboxCount( domain ),
-				} ) }
-				{ showExternalControlPanelLink && <Gridicon icon="external" /> }
+			<Button compact primary href={ setupMailboxUrl }>
+				{ translate( 'Set Up Mailbox' ) }
```

Comment:
I've decided to always show the same CTA even if there are several unused mailboxes since the target page only supports setting up one mailbox.

Reply:
Can we adjust this to "Set up mailbox" to match the task on the setup checklist?
```suggestion
				{ translate( 'Set up mailbox' ) }
```